### PR TITLE
Fast travis

### PR DESCRIPTION
**Pull request by RlonRyan**
At least in theory...

### CODE REVIEW

**Comment by RlonRyan** [comment]
The first run is always slow as everything has to be re-downloaded.

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+script:
+  - ./gradlew build -S
+  # So the cache doesn't get repacked every time.
+  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+
+# Cache libs, as to improve speed.
+cache:
+  directories:
+    # The libs is up to debate...
+    - ./libs
```
Comment (luacs1998) [inline .travis.yml]: Don't cache libs

**Comment by luacs1998** [comment]
Protip: travis allows the user (in this case me) to re-run a build. I'll just keep re-running it until I'm satisfied with the speed.

**Comment by RlonRyan** [comment]
Ah but that second commit ought to be squashed anyways...

**Comment by RlonRyan** [comment]
Why was that last one so slow...?

File: .travis.yml (hunk 1 above)
Comment (RlonRyan) [inline .travis.yml]: Got it.

**Comment by RlonRyan** [comment]
So what's the word?

**Comment by luacs1998** [comment]
Still can't outrun jenkins...

**Pull request merged**